Sources/MessagingPush/PushContent.swift

```diff
@@ -119,5 +128,6 @@ struct CioPushPayload: Codable {
     struct Push: Codable, AutoLenses {
         let link: String?
         let image: String?
+        let cioHandleBadge : Bool?
```

Comment:
seems a little restrictive, I imagined we'd want some representation of 
```
action -> one of `incr`, `decr`, `set` or `clear` // to increment, decrement, or set discrete values
value -> an integer
```
(note: this is just off the top of my head, not canon - feel free to play with a better API than this - for example, we may not need a decr-type action if we instead had `incr` (or a better named thing) support positive and negative numbers, and depending on what `set to zero` is interpreted as by the os, a `clear` may not be necessary either?)

also I've seen on my device number-less badges - how does that work?